src/main/java/appeng/me/storage/CellInventory.java

```diff
@@ -442,21 +444,58 @@ protected void loadCellItems()
 
 		final int types = (int) this.getStoredItemTypes();
 
-		for( int x = 0; x < types; x++ )
+		for( int slot = 0; slot < types; slot++ )
 		{
-			final ItemStack t = ItemStack.loadItemStackFromNBT( this.tagCompound.getCompoundTag( itemSlots[x] ) );
-			if( t != null )
+			NBTTagCompound compoundTag = this.tagCompound.getCompoundTag( itemSlots[slot] );
+			int stackSize = this.tagCompound.getInteger( itemSlotCount[slot] );
+			loadCellItem( compoundTag, stackSize );
+		}
+
+		// cellItems.clean();
+	}
+
+	private void loadCellItem( NBTTagCompound compoundTag, int stackSize )
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "loadCellItem" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)